README.md

```diff
@@ -44,12 +44,18 @@ cat iris.output/step_03
 
 If it worked, you should see a JSON representation of 4 versions of a decision tree for classifying irises.
 
-To use brushfire as a jar in your own project, add the following to your POM file:
+To use brushfire in your own SBT project, add the following to your `build.sbt`:
+
+```scala
+libraryDependencies += "com.stripe" %% "brushfire" % "0.4.0"
+```
+
+To use brushfire as a jar in your own Maven project, add the following to your POM file:
 
 ```
 <dependency>
   <groupId>com.stripe</groupId>
-  <artifactId>brushfire</artifactId>
+  <artifactId>brushfire_${scala.binary.version}</artifactId>
```

Comment:
The readme should probably just have `_2.11`, I think that's clearer. But we haven't published a scala-specific one anywhere.